Invalid reflect call in configuration/configuration.go:setFieldValue

https://github.com/haproxytech/client-native/blob/v2.0.2/configuration/configuration.go#L1485-L1502

in `setFieldValue` `field.Elem().Value()` is called to get the value of the frontends `UniqueIdHeader` field. This field is type string, so calling `Elem()` on it results in a panic.

```
panic: reflect: call of reflect.Value.Elem on string Value [recovered]
        panic: reflect: call of reflect.Value.Elem on string Value

goroutine 9 [running]:
testing.tRunner.func1.1(0x184a180, 0xc000465220)
        /usr/local/Cellar/go/1.14.2_1/libexec/src/testing/testing.go:940 +0x4dc
testing.tRunner.func1(0xc000165200)
        /usr/local/Cellar/go/1.14.2_1/libexec/src/testing/testing.go:943 +0x317
panic(0x184a180, 0xc000465220)
        /usr/local/Cellar/go/1.14.2_1/libexec/src/runtime/panic.go:975 +0x41b
reflect.Value.Elem(0x1826260, 0xc000d5c138, 0x198, 0x0, 0x0, 0x0)
        /usr/local/Cellar/go/1.14.2_1/libexec/src/reflect/value.go:820 +0x3b5
github.com/haproxytech/client-native/v2/configuration.setFieldValue(0x18ed2bb, 0x8, 0xc00015d241, 0x9, 0x1809847, 0xe, 0x1826260, 0xc000d5c138, 0x198, 0xc000ff2450, ...)
        /Users/z003tt1/go/pkg/mod/github.com/haproxytech/client-native/v2@v2.0.2/configuration/configuration.go:1496 +0xdfde
github.com/haproxytech/client-native/v2/configuration.CreateEditSection(0x18d3580, 0xc000d5c000, 0x18ed2bb, 0x8, 0xc00015d241, 0x9, 0xc000ff2450, 0x0, 0x0)
        /Users/z003tt1/go/pkg/mod/github.com/haproxytech/client-native/v2@v2.0.2/configuration/configuration.go:830 +0x2d7
github.com/haproxytech/client-native/v2/configuration.(*Client).createSection(0xc00012ca80, 0x18ed2bb, 0x8, 0xc00015d241, 0x9, 0x18d3580, 0xc000d5c000, 0xc0010372f0, 0x24, 0x0, ...)
        /Users/z003tt1/go/pkg/mod/github.com/haproxytech/client-native/v2@v2.0.2/configuration/configuration.go:1705 +0x564
github.com/haproxytech/client-native/v2/configuration.(*Client).CreateFrontend(0xc00012ca80, 0xc000d5c000, 0xc0010372f0, 0x24, 0x0, 0x0, 0x0)
        /Users/z003tt1/go/pkg/mod/github.com/haproxytech/client-native/v2@v2.0.2/configuration/frontend.go:117 +0x1a5
```

--- a/configuration/configuration.go
+++ b/configuration/configuration.go
@@ -1495,7 +1495,7 @@ func setFieldValue(section parser.Section, sectionName string, fieldName string,
 			return nil
 		}
 		d := types.UniqueIDHeader{
-			Name: field.Elem().String(),
+			Name: field.String(),
 		}
 		if err := p.Set(section, sectionName, "unique-id-header", &d); err != nil {
 			return err

--- a/configuration/frontend_test.go
+++ b/configuration/frontend_test.go
@@ -145,6 +145,7 @@ func TestCreateEditDeleteFrontend(t *testing.T) {
 		BindProcess:          "4",
 		Logasap:              "disabled",
 		UniqueIDFormat:       "%{+X}o_%fi:%fp_%Ts_%rt:%pid",
+		UniqueIDHeader:       "X-Unique-Id",
 	}
 
 	err := client.CreateFrontend(f, "", version)

